vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/test/grid-connector.test.ts

```diff
@@ -73,7 +82,7 @@ describe('grid connector', () => {
 
     beforeEach(async () => {
       // Use a smaller page size for testing
-      grid.pageSize = 25;
+      grid.pageSize = 10;
```

Comment:
Had to adjust these tests to ensure that at least two pages fit within the viewport. Previously, this wasn't the case, and the tests passed because the requests created by the clear method weren't cancelled.